README.md

```diff
@@ -18,14 +18,16 @@ To run the REPL just enter the following in the source code directory
 
 and start typing REPL commands there. The syntax of the REPL commands is
 
-    <repl-command> := <term> | <binding>
+    <repl-command> := <term> | <binding> | <loading>
     <term> ::= <variable>
              | <numeric-literal>
              | <character-literal>
              | <string-literal>
              | <function>
              | <application>
     <binding> ::= <variable> := <term>
+    <loading> ::= load <module-path>
+    <module-path> := [a-zA-Z][a-zA-Z0-9/]*
```

Comment:
I'm not sure it would be better. I mean it's hard for me to call spaces or non-english letters in for instance java package names a good practice. I don't mind to support that though. But only as a separate issue, and only with the strong justification why it is really needed for someone. So go ahead and file one. We can continue discussing this there.